srv/components/system/config/service_user.sls

```diff
@@ -19,6 +19,7 @@
 
     {% set base_dir = '/opt/seagate/users' %}
     {% set user_data = salt['pillar.get']('system:service-user') %}
+    {% set roles = salt['pillar.get']('system:'+grains["id"]+':roles') %}
```

Comment:
spaces around a binary operator will help, please

Reply:
comment is addressed